src/components/MediaSettings/MediaSettings.vue

```diff
@@ -672,7 +666,21 @@ export default {
 
 		setRecordingConsentGiven(value) {
 			this.$emit('update:recording-consent-given', value)
-		}
+		},
+
+		handleAudioInputIdChange(audioInputId) {
+			this.audioInputId = audioInputId
+			if (audioInputId !== null) {
+				this.updatePreferences('audioinput')
+			}
```

Comment:
Unlikely there will be a wrong value / undefined, but `null` and other exceptions are already handled by function, so should be safe to call it directly:

```suggestion
			this.updatePreferences('audioinput')
```